### DISCUSSION TITLE
How to use cordova-plugin-file?

### DISCUSSION THREAD

**Discussion by realtebo** [quasarframework/quasar#15177] [CLI - Capacitor / Cordova modes]
I would like to use [Cordova Plugin File](https://cordova.apache.org/docs/en/11.x/reference/cordova-plugin-file/index.html) because I need to download and save some picture for later offline access.

I do not know what to import and how to create a watch / computed to access `cordova.file.dataDirectory`.

Is there any example?

**Comment by metalsadman** [comment]
follow the lib's installation, you can't watch it as it's not reactive.
